.changeset/fresh-deers-brake.md

```diff
@@ -0,0 +1,7 @@
+---
+'lit': patch
+'lit-html': patch
+'@lit/reactive-element': patch
```

Comment:
Was lit-element built?

Reply:
Oops yes thank you, I also needed lit-element. Done.